dataplane/standalone/apigen/protogen/protogen.go

```diff
@@ -55,21 +61,47 @@ func Generate(doc *docparser.SAIInfo, sai *saiast.SAIAPI) (map[string]string, er
 	return files, nil
 }
 
+func rangeInOrder[T any](m map[string]T, pred func(key string, val T) error) error {
+	list := []*struct {
+		key string
+		val T
+	}{}
+	for k, v := range m {
+		list = append(list, &struct {
+			key string
+			val T
+		}{
+			key: k,
+			val: v,
+		})
+	}
+	sort.SliceStable(list, func(i, j int) bool {
+		return list[i].key < list[j].key
+	})
+	for _, e := range list {
+		if err := pred(e.key, e.val); err != nil {
+			return err
+		}
+	}
+	return nil
```

Comment:
What do you think about
```
keys := maps.Keys(m)
slices.Sort(keys)
for _, k := range keys {
}
```
https://pkg.go.dev/golang.org/x/exp/maps#Keys

Reply:
done